### DISCUSSION TITLE
ExpandableSearch and Tooltip

### DISCUSSION THREAD

**Discussion by gcattan** [carbon-design-system/carbon#15160] [Help]
I would like to put a tooltip on an expandable search. Is there a recommended way to do it with carbon? I don't see a `tooltip` props inside the search component, and the tooltip definition doesn't seems to work well when expanding.
Any idea?

**Comment by laurenmrice** [comment]
Hi @gcattan, could you share a screenshot of a visual of your usecase of where you are using the expandable search? It can help us provide better feedback. Thanks!

**Reply by gcattan** [reply]
Please find below two screenshots:

> ![image](https://github.com/carbon-design-system/carbon/assets/6229031/e38bd2b1-7c92-41f0-aa6b-2b9b16847fda)


> ![image](https://github.com/carbon-design-system/carbon/assets/6229031/af13ea60-cf39-4b61-bc21-1f0bc45191f5)
